justfile

```diff
@@ -35,11 +35,7 @@ clippy-fix:
         -A clippy::todo \
         -A clippy::unimplemented \
         -A clippy::indexing_slicing
-    @echo "Running cargo clippy with automatic fixes on potentially dirty code..."
-    cargo +{{RUSTV}} clippy --fix --allow-dirty --workspace --all-targets -- \
```

Comment:
Why are these checks removed?

Reply:
It was removed , as it clippy-fix was being called twice. Probably a mistake with a merge